src/core/highlight-vars.js

```diff
@@ -0,0 +1,29 @@
+// Module core/highlight-vars
+// Highlights occurrences of a <var> within a .algorithm on click
+// if a <var> is set only once (unused/undeclared), it's highlighted with class ".bug"
+
+import hlVars from "deps/text!core/css/var.css";
+const styleElement = document.createElement("style");
+styleElement.textContent = hlVars;
+styleElement.classList.add("removeOnSave");
+document.head.appendChild(styleElement);
+
+export const name = "core/highlight-vars";
+
+export function run(conf, doc, cb) {
```

Comment:
Nit: new plugins don't require `(conf, doc, cb)`, you can just do: `run(){ ... }`. Also gets rid of the `cb()`.

Reply:
I tried using this new plugin syntax. but `document.respecIsReady` didn't resolve. Am I missing something?

Reply:
could be, send up your code (or put it in gist) and I can have a look.

Reply:
https://gist.github.com/sidvishnoi/dc57028a6703948ecd14a62a20317f43

Reply:
yeah, that's odd... it looks fine. Just stick to the old style for now, I think. Otherwise, throw in a couple of `console.log()`s.